postman/Node-Mongo-Collection.postman_collection.json

```diff
@@ -0,0 +1,4739 @@
+{
```

Comment:
There is already a postman collection built. If you want to make some edits. make on it. Why is there two postman collection files